accounts/views.py

```diff
@@ -53,6 +53,40 @@ def get_users(self, request):
         serializer = UserSerializer(users, many=True)
         return Response(serializer.data, status=200)
 
+    @action(detail=False, methods=['put'])
+    def demote_user(self, request):
```

Comment:
@ozifirebrand are we saying anybody can demote a user here?

Kindly check with Femi to show you how to include permissions.

This should reflect in all the methods that need a degree of permissions.